cookie-banner-rules-list.json

```diff
@@ -1,6 +1,15 @@
 {
   "$schema": "./CookieBannerRuleList.schema.json",
   "data": [
+    {
```

Comment:
Please add the rule at the bottom of the file.

Reply:
updated accordingly